src/extension.ts

```diff
@@ -195,17 +208,18 @@ class WikilinkReferenceCodeLensProvider implements CodeLensProvider {
     document: TextDocument,
     _: CancellationToken
   ): CodeLens[] | Thenable<CodeLens[]> {
+    loadDocConfig();
```

Comment:
Ensuring that loadDocConfig is called even when CodeLens executes before other extension methods